app/forms/Validation/Validation.scala

```diff
@@ -21,7 +21,7 @@ object Validation {
   val euVatNumberPattern = """^[A-Za-z\d\*\+]{1,12}$"""
   val postcodePattern = """^[A-Za-z0-9 ]{0,100}$"""
   val commonTextPattern = """^(?!^[’'"])(?:[A-Za-z0-9À-ÿ \!\)\(.,_/’'"&-]|[’'"](?=[A-Za-z0-9À-ÿ \!\)\(.,_/’'"&-]))*[A-Za-z0-9À-ÿ \!\)\(.,_/’'"&-](?<![’'"]$)$"""
-  val bankAccountNamePattern = """^[A-Za-z0-9/\-?:().,'+ ]*$"""
+  val bankAccountNamePattern = """^[A-Za-z0-9\/\-?:().,'+ ]*$"""
```

Comment:
The escaping here, was it because you tested using a forward slash `/` and it didn't allow you to use?

Reply:
I put the regex in a regex checker and it said that the pattern contains one or more errors "/ An unescaped delimiter must be escaped; in most languages with a backslash (\)" not sure if it was an issue or not but when I added the backslash I was able to test the regex in the checker.

Reply:
can we test in code? I never know if it needs to be escaped in code or not